components/technology/modules/ROOT/pages/block-rotation.adoc

```diff
@@ -0,0 +1,47 @@
+= Block Rotation
+
+== Introduction
+
+SKALE Chain Block rotation allows to limit the space on hard drive which is occupied by blocks. The current default value is 12.6GB.
```

Comment:
Suggestion: SKALE Chain Block Rotation -> Capital R in Rotation

Reply:
fixed